journals/apps/journals/forms.py

```diff
@@ -33,3 +32,31 @@ class UsersJournalAccessForm(forms.Form):
         label=_("Users' CSV file"),
         help_text=_(".csv file with single column consists of only usernames"),
     )
+
+
+class JournalEditForm(forms.ModelForm):
+    """
+        Journal Edit form for wagtail CMS
+    """
+    status = forms.BooleanField(label=_('Status'), required=False)
+
+    class Meta:
+        model = Journal
+        exclude = ('access_length', 'organization')
+
+
+class JournalCreateForm(forms.ModelForm):
+    """
+        Journal Create form for wagtail CMS
+    """
+    price = forms.FloatField(label=_('Price'))
+    currency = forms.CharField(label=_('Currency'))
+
+    class Meta:
+        model = Journal
+        fields = '__all__'
```

Comment:
Do we need this `fields = '__all__'`?

Reply:
yes, well are adding all fields with two additional fields , mentioned above